src/pip/_internal/operations/prepare.py

```diff
@@ -16,26 +17,27 @@
 )
 from pip._internal.utils.hashes import MissingHashes
 from pip._internal.utils.logging import indent_log
-from pip._internal.utils.misc import display_path, normalize_path
+from pip._internal.utils.misc import display_path, normalize_path, call_subprocess
+from pip._internal.utils.ui import open_spinner
 from pip._internal.vcs import vcs
 
 logger = logging.getLogger(__name__)
 
 
-def make_abstract_dist(req):
+def make_abstract_dist(req_to_install):
```

Comment:
I prefer the `req` spelling; it's shorter and more to the point. Plus, this is noise in the PR diff.

Reply:
Will do.